osquery/tables/networking/aws_metadata.cpp

```diff
@@ -0,0 +1,238 @@
+/*
+ *  Copyright (c) Jean-Francois Dive <jfdive@gmail.com>
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. 
+ *
+ */
+
+#include <string>
+
+#include <boost/network/protocol/http/client.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/noncopyable.hpp>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+namespace pt = boost::property_tree;
+namespace http = boost::network::http;
+
+namespace osquery {
+namespace tables {
+
+/**
+ * @brief SuperClass for AWS metadata accessors
+ */
+class AwsData : public boost::noncopyable {
+  protected:
+  /**
+   * @brief SQL type for the value
+   */
+  ColumnType sqlType;
```

Comment:
For private/protected member data the project prefers snake_case with a trailing `_` please :)